Automattic/wp-calypso
# Happy-Blocks: Implement Global Header i2

This implemented the Global Header i2.

## Testing Instructions

1. cd into `apps/happy-blocks`.
2. Run `yarn dev --sync`.
3. Sandbox wordpress.com
4. Go to wordpress.com/support.
5. Make sure the Header matches the one in the Figma. 
6. Reside the window, the search bar should remain aligned with the 3rd block at all sizes until mobile kicks in.

![image](https://user-images.githubusercontent.com/17054134/233663395-05cc67d1-a515-4138-aa25-8a05d5de6e51.png)
